### ISSUE TITLE
Emeter JSON format varies between models/firmware, improvement suggested.

### ISSUE THREAD

**Issue by cdickens77** [anthturner/TPLinkSmartDevices#11]
The JSON emeter values exposed by a current model KP115 Smart-Plug Mini match the field maps in PowerData.cs:

```
myPlug.Model = "KP115(US)"
myPlug.SoftwareVersion = "1.0.17 Build 210506 Rel.075231"
myPlug.CurrentPowerUsage._powerData = "{{
  "current_ma": 1542,
  "voltage_mv": 120356,
  "power_mw": 181156,
  "total_wh": 192,
  "err_code": 0
}}"
```

However, ironically, the latest firmware for HS110 according ot my KASA app does not:

```
myPlug.Model = "HS110(US)"
myPlug.SoftwareVersion = "1.2.6 Build 200727 Rel.121701"
myPlug.CurrentPowerUsage._powerData = "{{
  "current": 0.013112,
  "voltage": 122.613419,
  "power": 0,
  "total": 0.826,
  "err_code": 0
}}"
```

The attached PowerData.cs adds coalesce code to allow both formats.

[PowerData.zip](https://github.com/anthturner/TPLinkSmartDevices/files/7117581/PowerData.zip)

**Comment by cdickens77** [comment]
I just realized the informational .ToString method actually had a bug in it originally where the mW value was converted twice to W causing invalid readings for devices using the power_mw format. Revised version attached tested with both KP115 and HS110:

[PowerData2.zip](https://github.com/anthturner/TPLinkSmartDevices/files/7117664/PowerData2.zip)

**Comment by anthturner** [comment]
Thanks for the bug report! Can you open a pull request? I'd be happy to accept.

**Comment by cdickens77** [comment]
Submitted.  Thanks!

**Comment by cdickens77** [comment]
Resolved by Pull #12. Closing.
